src/ontology/cl-edit.owl

```diff
@@ -32388,6 +32390,7 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:12429182") Annotation(ob
 AnnotationAssertion(rdfs:label obo:CL_4042021 "neuronal-restricted precursor"@en)
 SubClassOf(obo:CL_4042021 obo:CL_0000123)
 SubClassOf(obo:CL_4042021 obo:CL_0011026)
+SubClassOf(obo:CL_4042021 obo:CL_2000097)
```

Comment:
Is this a stray edit on a different term?  Looks a bit odd.

Reply:
all done